src/client/Client.cc

```diff
@@ -3543,9 +3549,8 @@ void Client::check_caps(Inode *in, unsigned flags)
     if (!revoking && unmounting && (cap_used == 0))
       goto ack;
 
-    if (wanted == cap.wanted &&         // mds knows what we want.
```

Comment:
Question regarding this change (7922a292e9d1c69cc2298b0126ae3a16b65c2763): does this mean the MDS will always wait for the revoke of issued caps even if the wanted caps do not include the caps? Is `wanted` really just for communicating the desired caps by the client (i.e. "MDS please give me all of these caps if possible")?